modules/util.py

```diff
@@ -68,7 +68,7 @@ def is_tag_in_torrent(check_tag, torrent_tags, exact=True):
             return tags_to_remove
 
 
-def format_stats_summary(stats: dict, config) -> list[str]:
+def format_stats_summary(stats: dict, config):
```

Comment:
```suggestion
def format_stats_summary(stats: dict, config) -> list[str]:
```
Please keep the type annotations